### PR TITLE
NMS-13694: Migrate Discourse charts article into main docs

### PR DESCRIPTION
**Pull request by Bonrob2**
### All Contributors

* [ ] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13694

### CODE REVIEW

File: docs/modules/operation/pages/admin/webui/dashboard.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 = Dashboard
 
-In Network Operation Centers NOC an overview about issues in the network is important and often described as dashboards.
+In Network Operations Centers NOC an overview about issues in the network is important and often described as dashboards.
```
Comment (mmahacek) [inline docs/modules/operation/pages/admin/webui/dashboard.adoc:4]:
```suggestion
In Network Operations Centers (NOCs) an overview of issues in the network is important and often presented as dashboards.
```

File: docs/modules/operation/pages/admin/webui/opsboard/dashlet/grafana.adoc [hunk 2]
```diff
@@ -1,23 +1,27 @@
 
 = Grafana
 
-This dashlet shows a Grafana dashboard for a given time range.
+This dashlet displays a Grafana dashboard for a given time range in the xref:admin/webui/opsboard/introduction.adoc#opsboard-config[Ops Board].
 The <<admin/webui/grafana-dashboard-box.adoc#webui-grafana-dashboard-box, Grafana Dashboard Box>> configuration defined in the `opennms.properties` file is used to access the Grafana instance.
```
Comment (mmahacek) [inline docs/modules/operation/pages/admin/webui/opsboard/dashlet/grafana.adoc:5]:
```suggestion
The <<admin/webui/grafana-dashboard-box.adoc#webui-grafana-dashboard-box, Grafana Dashboard Box>> configuration is used to access the Grafana instance.
```

File: docs/modules/operation/pages/admin/webui/opsboard/introduction.adoc [hunk 3]
```diff
@@ -2,56 +2,53 @@
...
 
-In a network operation center (NOC) you can use the ops board to visualize monitoring information.
-The monitoring information for various use-cases are arranged in configurable dashlets.
-To address different user groups it is possible to create multiple ops boards.
+An operator board helps network operations centers (NOCs) visualize monitoring information.
+Use configurable dashlets to arrange how to display different types of information (alarms, maps, KSC reports, and so on) on the board.
+You can also create multiple ops boards, customized for different user groups.
 
-There are two visualisation components to display dashlets:
+There are two visualization components to display dashlets:
```
Comment (mmahacek) [inline docs/modules/operation/pages/admin/webui/opsboard/introduction.adoc:9]:
```suggestion
There are two visualization modes to display dashlets:
```

**Review by mmahacek — CHANGES_REQUESTED** [review]
This probably should be retargeted to `foundation-2021`.

**Comment by Bonrob2** [comment]
> This probably should be retargeted to `foundation-2021`.

Yes, thanks. I meant to do that.

**Pull request closed without merging**